src/components/pages/error/Error.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useNavigate } from "react-router-dom"
+
+export default function Error() {
+
+  const Navigate = useNavigate();
+
+  const handleLogin= (e) => {
```

Comment:
```suggestion
  const redirectToLoginPage= (e) => {
```

Quitte à mettre un nom, préfère un nom explicite et logique 😉